src/Common/Hooks/useUrlFilters/useUrlFilters.ts

```diff
@@ -126,6 +127,7 @@ const useUrlFilters = <T = string, K = unknown>({
 
     const clearFilters = () => {
         history.replace({ search: '' })
+        localStorage.setItem(localStorageKey, '')
```

Comment:
shouldn't we check if the localStorageKey exists? Better would be create a internal function for setting/getting/removing the item